src/rest_vol_attr.c

```diff
@@ -114,7 +114,7 @@ RV_attr_create(void *obj, const H5VL_loc_params_t *loc_params, const char *attr_
 
     /* If this is a call to H5Acreate_by_name, locate the real parent object */
     if (H5VL_OBJECT_BY_NAME == loc_params->type) {
-        htri_t search_ret;
+        hbool_t search_ret;
```

Comment:
Since RV_find_object_by_path doesn't appear to have changed, all of these places where `htri_t search_ret` was changed to `hbool_t search_ret` should just remain as `htri_t search_ret` to match the prototype for RV_find_object_by_path. That, or RV_find_object_by_path should be changed to return an herr_t so we can detect failure, and then accept an `hbool_t` parameter through which it returns the existence status of the object being looked for. In the "link exists" and "attribute exists" callbacks, those arguments being passed in DO need to change from `htri_t` to `hbool_t`, but the `search_ret` variables in those callbacks still need to be `htri_t`. Eventually htri_t will probably go away, but for now